### PR TITLE
GitHub image preview: better compatibility with other extensions

### PR DESCRIPTION
**Pull request by kidonng**
I just found this script and it works like charm! I mean, without other extensions in the way...

This pull request fixes some compatibility issues with other browser extensions:

- Refined GitHub's [`quick-file-edit`](https://github.com/refined-github/refined-github/blob/0741fc984903b14ae1c3bceabbbae0ea05f22d8d/source/features/quick-file-edit.tsx) feature, which is a one line fix
- File icon extensions (in general), which also brings some style change

Now icons look good in dark mode:

image

With https://github.com/Claudiohbsantos/github-material-icons-extension:

image

I'm aiming for a minimal diff in case you want to argue about e.g. the variable name a little bit.

### CODE REVIEW

File: github-image-preview.user.js [hunk 1]
```diff
@@ -185,7 +186,7 @@
 			let content = "";
 			// not every submodule includes a link; reference examples from
 			// see https://github.com/electron/electron/tree/v1.1.1/vendor
-			const el = $("a", row) || $("div[role='rowheader'] span[title]", row);
+			const el = $("div[role='rowheader'] a, div[role='rowheader'] span[title]", row);
```
Comment (kidonng) [inline github-image-preview.user.js:189]: This is the one line fix for Refined GitHub

File: github-image-preview.user.js [hunk 2]
```diff
@@ -215,28 +216,16 @@
...
-					// get file name + extension
-					const str = url.substring(url.lastIndexOf("/") + 1, url.length);
-					// don't include extension for folders, or files with no extension,
-					// or files starting with a "." (e.g. ".gitignore")
-					content += (!noExt && str.indexOf(".") > 0) ?
-						"<h4 class='ghip-file-type'>" +
-						str
-							.substring(str.lastIndexOf(".") + 1, str.length)
-							.toUpperCase() +
-						"</h4></span>" : "</span>";
```
Comment (kidonng) [inline github-image-preview.user.js:239]: This does not look good with file icon extensions, so I dropped it. However, it makes it a little worse for those who _don't_ use them. I believe the tradeoff is worth it, let me know your thoughts.

File: github-image-preview.user.js (hunk 1 above)
Comment (Mottie) [reply github-image-preview.user.js:189]: Thanks!

File: github-image-preview.user.js (hunk 2 above)
Comment (Mottie) [reply github-image-preview.user.js:239]: Yeah, I think you're right.. I'm going to go ahead and merge your work, but I'm going to tweak it a bit to show the full file name. Thanks again!

**Pull request merged**